### PR TITLE
Utf8Proc : bump, soname changed

### PR DESCRIPTION
**Pull request by extrowerk**

### CODE REVIEW

File: dev-libs/libutf8proc/libutf8proc2-2.4.0.recipe [hunk 1]
```diff
@@ -0,0 +1,66 @@
+SUMMARY="A library for processing UTF-8 encoded Unicode strings"
+DESCRIPTION="utf8proc is a small, clean C library that provides Unicode \
+normalization, case-folding, and other operations for data in the UTF-8 \
+encoding, supporting Unicode version 7.0."
+HOMEPAGE="https://julialang.org/utf8proc/"
+COPYRIGHT="2006-2019 Public Software Group"
```
Comment (alaviss) [inline dev-libs/libutf8proc/libutf8proc2-2.4.0.recipe]:
```suggestion
COPYRIGHT="2006-2019 Public Software Group
	2014-2019 Steven G. Johnson, Jiahao Chen, Tony Kelman, Jonas Fonseca, \
	and other contributors listed in the git history"
```

From https://github.com/JuliaStrings/utf8proc/blob/v2.4.0/LICENSE.md#utf8proc-license

**Comment by alaviss** [comment]
There seems to be a test suite [attached](https://github.com/JuliaStrings/utf8proc/blob/v2.4.0/Makefile#L162), please add a `TEST()` function for this.

File: dev-libs/libutf8proc/libutf8proc2-2.4.0.recipe (hunk 1 above)
Comment (extrowerk) [reply dev-libs/libutf8proc/libutf8proc2-2.4.0.recipe]: The cmake build system have no support for test yet.

**Comment by waddlesplash** [comment]
@extrowerk can you submit the GNUInstallDirs patch upstream while you're at it?

**Pull request merged**